cosmotech/coal/store/store.py

```diff
@@ -3,19 +3,23 @@
 
 import pyarrow
 from adbc_driver_sqlite import dbapi
-from cosmotech.orchestrator.utils.singleton import Singleton
 
 from cosmotech.coal.utils.logger import LOGGER
 
 
-class Store(metaclass=Singleton):
+class Store:
 
     @staticmethod
     def sanitize_column(column_name: str) -> str:
         return column_name.replace(" ", "_").lower()
 
-    def __init__(self, reset=False):
-        self.store_location = pathlib.Path(os.environ.get("CSM_PARAMETERS_ABSOLUTE_PATH", ".")) / ".coal/store"
+    def __init__(
+        self,
+        reset=False,
```

Comment:
For a singleton I've understood but now it seem more confusing than useful. looks like a gimmick not to call the reset function.